app/services/edition_presenter_factory.rb

```diff
@@ -0,0 +1,16 @@
+module EditionPresenterFactory
```

Comment:
I think it'd be better in this instance to keep things consistent. I believe even `extend self` might be preferable to `module_function` here.